### DISCUSSION TITLE
QUESTION: how do you add extensions from terminal?

### DISCUSSION THREAD

**Discussion by profetik-777** [castrojo/ublue#46] [General]
I am using this script to learn more about scripts/gnome in general. 

Wanted to know how do you "source" the gnome extensions - when I try to add the sound selector I get 

"gnome-shell-extension-sound-output-device-chooser" not found. 

But I can't find where you add the gnome extensions back end from....

**Comment by castrojo** [comment]
I find the list of packages by doing a `dnf search` in a fedora toolbox and then using that name. Alternatively, I include the "Extensions Manager" application that will just grab it from extensions.gnome.org instead of doing it via the package manager, that's how I manage most of my extensions.
